samcli/lib/utils/stream_writer.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, stream, auto_flush=False):
     def stream(self):
         return self._stream
 
-    def write(self, output, encode=False):
+    def write(self, output, encode=False, write_to_buffer=True):
```

Comment:
Just wondering, are there other places where this is called with `write_to_buffer=True`?

Reply:
I added `write_to_buffer` to keep all the previous code paths the same. I was seeing different behavior when writing to `sys.stdout.buffer` and `sys.stdout`. I wanted to write to `sys.stdout` for the output of the invoke but leave everything else writing to the buffer. So I added this option to the write call, defaulted to the previous behavior of writing to the buffer, and then passing is `False` for my specific case. 

Ideally, I would have wanted to change everything but that was a much bigger change and coming from all the docker upgrade issues, I went the least impactful route.